benchmarks/sharded-bm/cases/log_patch.json

```diff
@@ -1,5 +1,5 @@
 {
     "rust_log": "transaction-generator=info,tokio_reactor=info,config=info,near=info,stats=info,telemetry=info,db=info,delay_detector=info,near-performance-metrics=info,state_viewer=info,warn",
     "verbose_module": null,
-    "opentelemetry": null
-  }
\ No newline at end of file
+    "opentelemetry": "client=debug,chain=debug,stateless_validation=debug,info"
```

Comment:
Would it make sense to set this configuration on the fly, when tracing is enabled? For instance in `edit_log_config`

Or having this set to `debug` does nothing if `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` is not set?